user/src/Components/RequestOverrideButton.tsx

```diff
@@ -0,0 +1,26 @@
+import {useState} from 'react';
+import useInitiateRequestOverride from "../hooks/useInitiateRequestOverride.tsx";
+
+export default function RequestOverrideButton({requestId, requestDate}: { requestId: string, requestDate: string }) {
+    const {initiateRequestOverride, loading, error} = useInitiateRequestOverride();
+    const [status, setStatus] = useState('');
+
+    const handleClick = async () => {
+        const dto = {approvalStatus: false, date: requestDate, state: 'MANUAL_OVERRIDE_REQUESTED'};
+        const updatedRequest = await initiateRequestOverride(requestId, dto);
+        if (updatedRequest) {
+            setStatus(`Request updated to state: ${updatedRequest.state}`);
```

Comment:
Nit: This is very developer speak, and not how we should show stuff in an app. It would be more appropriate to put something like "Request submitted" or "Pending admin review". I get that its good for debugging but not something we should have showing on the page.